app/src/main/java/com/eventyay/organizer/core/event/create/UpdateEventFragment.java

```diff
@@ -113,6 +114,12 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
         setHasOptionsMenu(true);
 
+        if(BuildConfig.FLAVOR.equals("fdroid")) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/app/fossasia/open-event-orga-app/pullRequest?prid=4593769)